packages/core/src/stylable-processor.ts

```diff
@@ -136,6 +141,12 @@ export const processorWarnings = {
     MISSING_SCOPING_PARAM() {
         return '"@st-scope" missing scoping selector parameter';
     },
+    MISSING_KEYFRAMES_PARAM() {
```

Comment:
This should probably be `MISSING_KEYFRAMES_NAME`, with the texts using `name` instead of `param`.

Param originates from the at-rule side of things, and not from the keyframes/animation semantics.